enrich-points-with-boundaries/geodatabase/src/main/scala/com/precisely/bigdata/sample/spark/GeodatabaseEnrichment.scala

```diff
@@ -36,6 +36,11 @@ object GeodatabaseEnrichment {
     val downloadLocation = args(3)
 
     val sparkConf = new SparkConf().setIfMissing("spark.master", "local[*]")
+    var sparkMajorVersion = 2
+    sparkMajorVersion=org.apache.spark.SPARK_VERSION.split('=')(0).split('.')(0).toInt
```

Comment:
Do we need to to initialize  sparkMajorVersion  with 2 in case when we are determining the major version on line# 40?
and can we merge statements of line# 39 & 40?

Reply:
Done for all test driver classes.